CMakeLists.txt

```
@@ -108,14 +118,15 @@ list(FILTER studio-sources EXCLUDE REGEX "UTESTSCB.CPP$")
 target_sources(studio PRIVATE ${studio-sources})
 target_include_directories(studio
   PUBLIC $<TARGET_PROPERTY:engine,INCLUDE_DIRECTORIES>
-  "${PROJECT_SOURCE_DIR}/SRC")
+  "${PROJECT_SOURCE_DIR}/SRC"
+)
 target_link_libraries(studio PRIVATE engine)
 set_property(TARGET studio PROPERTY OUTPUT_NAME 3dmovie)
-#target_link_options(studio BEFORE PRIVATE $<$<LINK_LANG_AND_ID:CXX,MSVC>:/VERBOSE:LIB>)
-# This resolves the linker warning where multiple runtime libraries are linked in.
-target_link_options(studio
-  BEFORE PRIVATE
-    $<$<AND:$<LINK_LANG_AND_ID:CXX,MSVC>,$<CONFIG:DEBUG>>:/NODEFAULTLIB:libcmt.lib>)
+if (MSVC)
+  # NOTE(bjrkk): this enforces registry compatibility shims on Windows,
+  # thus letting the game actually read its own registry keys in HKEY_LOCAL_MACHINE.
+  target_link_options(studio BEFORE PRIVATE /MANIFESTUAC:NO)
+endif ()
```

```suggestion
target_link_options(studio BEFORE PRIVATE $<$<LINK_LANG__ID:CXX,MSVC>:/MANIFESTUAC:NO>)
```

I tend to recommend using generator expressions because then you can put the logic in at generation time, and also stack up multiple flags/expressions without worrying about hairy if-else chains.